### PR TITLE
401 feature update to keptn 1

### LINKED ISSUE #401: [FEATURE] Update to keptn 1.0
Todo:

- Update acebox to deploy keptn 1.0 
--- Keptn 1.0
--- Jmeter service (update url)
--- Dynatrace service for 1.0 test (sync with internal team)
--- Gitea provisioning service update (test if it works, Kristof has a branch with progress) https://github.com/keptn-sandbox/keptn-gitea-provisioner-service

- Update cloud automation tools  https://github.com/dynatrace-ace/cloud-automation-tools
--- Update endpoints and payloads to configure the project
--- Update endpoints to upload the resources 
--- Update endpoints to run a quality gate evaluation 
--- Run it as container (upload to dynatrace Ace dockerhub)

Update pipelines 
--- Update jenkins pipeline (docker) to run the new cloud automation tools
--- Update Gitlab (docker)

Update documentation
--- New implementation details and if there are any breaking changes from user perspective

Test 
--- End 2 end demos 
----- Quality gate jenkins
----- Quality gate Gitlab
----- Security gates

### PR DESCRIPTION
**Pull request by eemrdog**
Tested with demo-quality-gates-gitlab and demo-quality-gates-jenkins to see the successful keptn v1 evaluations

### CODE REVIEW

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml [hunk 1]
```diff
@@ -71,9 +71,9 @@ monaco_config_event_prod:
     CUSTOM_PROPERTIES: '{"Approved by": "ACE"}'
 
 init_cloudautomation:
-  image: dynatraceace/keptn-gitlab-runner:2.2
+  image: dynatraceace/cloud-automation-tools:latest
```
Comment (kristofre) [inline user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml]: @eemrdog never use `latest`, always pin to a specific version

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml [hunk 2]
```diff
@@ -121,28 +121,29 @@ dynatrace_deploy_event_staging:
...
   artifacts:
     paths:
-    - keptn.test.starttime
-    - keptn.test.endtime
+      - keptn.test.starttime
+      - keptn.test.endtime
 
 quality_gate:
-  image: dynatraceace/keptn-gitlab-runner:2.2
+  image: dynatraceace/cloud-automation-tools:latest
```
Comment (kristofre) [inline user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml]: same as above

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml (hunk 1 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-simplenode/files/repos/simplenode/.gitlab-ci.yml]: updated to 1.0.0

**Review by kristofre — APPROVED** [review]

File: user-skel/ansible/roles/app-simplenode/files/repos/simplenode/app.js [hunk 3]
```diff
@@ -283,9 +283,9 @@ var server = http.createServer(async function (req, res) {
 			// simply returns the build number as defined in BUILD_NUMBER env-variable which is specified
 			status = "Running build number: " + buildNumber + " Production-Mode: " + inProduction;
 			status += "\n\nHere some additional environment variables:";
-			status += "\nKEPT_PROJECT: " + keptn_project;
-			status += "\nKEPTN_STAGE: " + keptn_stage;
-			status += "\nKEPTN_SERVICE: " + keptn_service;
+			status += "\nKEPT_PROJECT: " + cloud_automation_project;
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/app-simplenode/files/repos/simplenode/app.js]: There's another KEPTN_PROJECT.

**Pull request merged**